### ISSUE TITLE
Compilation against postgresql master/PG 12 beta1?

### ISSUE THREAD

**Issue by jlrobins** [pgspider/sqlite_fdw#16]
When trying to compile against PG 12 Beta1 / current HEAD of postgresql master branch, am seeing what looks to be header file cycles. Anyone tried / trying against PG 12 beta, or can point me at a tool which I could use to detect

```
jlrobins:sqlite_fdw jlrobins$ make USE_PGXS=1
gcc -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Werror=vla -Wendif-labels -Wmissing-format-attribute -Wformat-security -fno-strict-aliasing -fwrapv -Wno-unused-command-line-argument -g -O2  -I. -I./ -I/Users/jlrobins/pg-devel/include/postgresql/server -I/Users/jlrobins/pg-devel/include/postgresql/internal -I/opt/local/include -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk  -I/opt/local/include/libxml2  -I/opt/local/include  -c -o option.o option.c
In file included from option.c:29:
In file included from /Users/jlrobins/pg-devel/include/postgresql/server/foreign/fdwapi.h:17:
/Users/jlrobins/pg-devel/include/postgresql/server/nodes/pathnodes.h:35:14: error: redefinition of 'CostSelector'
typedef enum CostSelector
             ^
/Users/jlrobins/pg-devel/include/postgresql/server/nodes/relation.h:35:14: note: previous definition is here
typedef enum CostSelector
             ^
In file included from option.c:29:
In file included from /Users/jlrobins/pg-devel/include/postgresql/server/foreign/fdwapi.h:17:
/Users/jlrobins/pg-devel/include/postgresql/server/nodes/pathnodes.h:37:2: error: redefinition of enumerator
      'STARTUP_COST'
        STARTUP_COST, TOTAL_COST
        ^
/Users/jlrobins/pg-devel/include/postgresql/server/nodes/relation.h:37:2: note: previous definition is here
        STARTUP_COST, TOTAL_COST
        ^
In file included from option.c:29:

....

fatal error: too many errors emitted, stopping now [-ferror-limit=]
20 errors generated.
make: *** [option.o] Error 1

```

Is this just me, or can others reproduce and / or has development against master / PG 12 started?

**Comment by mochizk** [comment]
Thanks for reporting the issue.
We do not support PG 12 yet.
We plan to support it, but for now could you use PG 9, 10 or 11?

**Comment by devrimgunduz** [comment]
ping. v12 is out soon.

**Comment by mochizk** [comment]
Thanks for the reminder!

I pushed PG 12 support to master.
I will release when PG 12 is released.

**Comment by mochizk** [comment]
Thanks for the packaging, I have just released as 1.2.0.
